src/tests/Microsoft.PowerFx.Connectors.Tests/Responses/AzureBlobStorage_Paging_Response1.json

```diff
@@ -0,0 +1,71 @@
+{
+  "value": [
+    {
+      "Id": "JTJmY29udGFpbmVyJTJmMC50eHQ=",
+      "Name": "0.txt",
+      "DisplayName": "0.txt",
+      "Path": "/container/0.txt",
+      "LastModified": "2023-07-31T13:25:28Z",
+      "Size": 0,
+      "MediaType": "text/plain",
+      "IsFolder": false,
+      "ETag": "\"0x8DB91C99B2E2450\"",
+      "FileLocator": "JTJmY29udGFpbmVyJTJmMC50eHQ=",
+      "LastModifiedBy": null
+    },
+    {
+      "Id": "JTJmY29udGFpbmVyJTJmMS50eHQ=",
+      "Name": "1.txt",
+      "DisplayName": "1.txt",
+      "Path": "/container/1.txt",
+      "LastModified": "2023-07-31T13:28:07Z",
+      "Size": 0,
+      "MediaType": "text/plain",
+      "IsFolder": false,
+      "ETag": "\"0x8DB91C9FA2324AC\"",
+      "FileLocator": "JTJmY29udGFpbmVyJTJmMS50eHQ=",
+      "LastModifiedBy": null
+    },
+    {
+      "Id": "JTJmY29udGFpbmVyJTJmMTAudHh0",
+      "Name": "10.txt",
+      "DisplayName": "10.txt",
+      "Path": "/container/10.txt",
+      "LastModified": "2023-07-31T13:28:07Z",
+      "Size": 0,
+      "MediaType": "text/plain",
+      "IsFolder": false,
+      "ETag": "\"0x8DB91C9FA6E2C84\"",
+      "FileLocator": "JTJmY29udGFpbmVyJTJmMTAudHh0",
+      "LastModifiedBy": null
+    },
+    {
+      "Id": "JTJmY29udGFpbmVyJTJmMTAwLnR4dA==",
+      "Name": "100.txt",
+      "DisplayName": "100.txt",
+      "Path": "/container/100.txt",
+      "LastModified": "2023-07-31T13:28:12Z",
+      "Size": 0,
+      "MediaType": "text/plain",
+      "IsFolder": false,
+      "ETag": "\"0x8DB91C9FD39DC92\"",
+      "FileLocator": "JTJmY29udGFpbmVyJTJmMTAwLnR4dA==",
+      "LastModifiedBy": null
+    },
+    {
+      "Id": "JTJmY29udGFpbmVyJTJmMTAwMC50eHQ=",
+      "Name": "1000.txt",
+      "DisplayName": "1000.txt",
+      "Path": "/container/1000.txt",
+      "LastModified": "2023-07-31T13:28:59Z",
+      "Size": 0,
+      "MediaType": "text/plain",
+      "IsFolder": false,
+      "ETag": "\"0x8DB91CA1915457C\"",
+      "FileLocator": "JTJmY29udGFpbmVyJTJmMTAwMC50eHQ=",
+      "LastModifiedBy": null
+    }    
+  ],
+  "nextLink": "https://tip1-shared-002.azure-apim.net/apim/azureblob/d95489a91a5846f4b2c095307d86edd6/v2/datasets/pfxdevstgaccount1/foldersV2/container?nextPageMarker=2!72!MDAwMDA4ITU0OTgudHh0ITAwMDAyOCE5OTk5LTEyLTMxVDIzOjU5OjU5Ljk5OTk5OTlaIQ--&useFlatListing=False",
```

Comment:
just double checking - this high-entropy url doesn't contain any secrets?

Reply:
nope, it's not a SAS token